pkg/otlp/metrics/metrics_translator_benchmark_test.go

```diff
@@ -428,3 +498,68 @@ func BenchmarkMapDeltaSumMetrics10000000(b *testing.B) {
 
 	benchmarkMapMetrics(metrics, b)
 }
+
+func BenchmarkMapSumRuntimeMetric(b *testing.B) {
+	for _, v := range inputTable {
+		b.Run(fmt.Sprintf("BenchmarkMapRuntimeMetricsHasMapping-%d", v.input), func(b *testing.B) {
+			benchmarkMapMetrics(createBenchmarkRuntimeMetric("process.runtime.go.goroutines", pmetric.MetricTypeSum, nil, v.input), b)
+		})
+		b.Run(fmt.Sprintf("BenchmarkMapRuntimeMetricsNoMapping-%d", v.input), func(b *testing.B) {
+			benchmarkMapMetrics(createBenchmarkRuntimeMetric("go.goroutines", pmetric.MetricTypeSum, nil, v.input), b)
+		})
+	}
+}
+
+func BenchmarkMapGaugeRuntimeMetricWithAttributesHasMapping(b *testing.B) {
+	attr := []runtimeMetricAttribute{{
+		key:    "generation",
+		values: []string{"gen1"},
+	}}
+
+	for _, v := range inputTable {
+		b.Run(fmt.Sprintf("BenchmarkMapGaugeRuntimeMetricWithAttributesHasMapping-%d", v.input), func(b *testing.B) {
+			benchmarkMapMetrics(createBenchmarkRuntimeMetric("process.runtime.dotnet.gc.heap.size", pmetric.MetricTypeGauge, attr, v.input), b)
+		})
+		b.Run(fmt.Sprintf("BenchmarkMapGaugeRuntimeMetricWithAttributesNoMapping-%d", v.input), func(b *testing.B) {
+			benchmarkMapMetrics(createBenchmarkRuntimeMetric("dotnet.gc.heap.size", pmetric.MetricTypeGauge, attr, v.input), b)
+		})
+	}
+}
+
+func BenchmarkMapGaugeRuntimeMetricWith10AttributesHasMapping(b *testing.B) {
+	var attr []runtimeMetricAttribute
+	for i := 1; i <= 10; i++ {
+		attr = append(attr, runtimeMetricAttribute{
+			key:    "generation",
+			values: []string{fmt.Sprintf("gen%d", i)},
+		})
+	}
+
+	for _, v := range inputTable {
+		b.Run(fmt.Sprintf("BenchmarkMapGaugeRuntimeMetricWith10AttributesHasMapping-%d", v.input), func(b *testing.B) {
+			benchmarkMapMetrics(createBenchmarkRuntimeMetric("process.runtime.dotnet.gc.heap.size", pmetric.MetricTypeGauge, attr, v.input), b)
```

Comment:
Can u create const for metrics Name `process.runtime.dotnet.gc.heap.size` ? It isn't clear to me which ones are going through translation logic which are not.